catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/builder/AttributeValueNormalizerTest.java

```diff
@@ -208,6 +208,60 @@ public void testInvalidRelativeFunctionToXml() {
     normalizer.normalizeForXml(PROPERTY_NAME, INVALID_RELATIVE_FUNCTION);
   }
 
+  @Test
+  public void testValidDuringRangeToXml() {
+    when(registry.lookup(eq(PROPERTY_NAME))).thenReturn(Optional.of(DATE_DESCRIPTOR));
+    assertThat(
+        normalizer.normalizeForXml(
+            PROPERTY_NAME, VALID_ISO_8601_DATE_STRING + "/" + VALID_ISO_8601_DATE_STRING),
+        is(equalTo(VALID_MS_SINCE_EPOCH_DATE_STRING + "/" + VALID_MS_SINCE_EPOCH_DATE_STRING)));
+  }
+
+  @Test
+  public void testValidOffsetDuringRangeToXml() {
+    when(registry.lookup(eq(PROPERTY_NAME))).thenReturn(Optional.of(DATE_DESCRIPTOR));
+    assertThat(
+        normalizer.normalizeForXml(
+            PROPERTY_NAME,
+            VALID_ISO_8601_DATE_STRING_WITH_OFFSET + "/" + VALID_ISO_8601_DATE_STRING_WITH_OFFSET),
+        is(
+            equalTo(
+                VALID_MS_SINCE_EPOCH_DATE_STRING_WITH_OFFSET
+                    + "/"
+                    + VALID_MS_SINCE_EPOCH_DATE_STRING_WITH_OFFSET)));
+  }
+
+  @Test
+  public void testValidOnlyFromDuringRangeToXml() {
+    when(registry.lookup(eq(PROPERTY_NAME))).thenReturn(Optional.of(DATE_DESCRIPTOR));
+    assertThat(
+        normalizer.normalizeForXml(PROPERTY_NAME, VALID_ISO_8601_DATE_STRING_WITH_OFFSET + "/"),
+        is(equalTo(VALID_MS_SINCE_EPOCH_DATE_STRING_WITH_OFFSET + "/")));
+  }
+
+  @Test
+  public void testValidOnlyToDuringRangeToXml() {
+    when(registry.lookup(eq(PROPERTY_NAME))).thenReturn(Optional.of(DATE_DESCRIPTOR));
+    assertThat(
+        normalizer.normalizeForXml(PROPERTY_NAME, "/" + VALID_ISO_8601_DATE_STRING_WITH_OFFSET),
+        is(equalTo("/" + VALID_MS_SINCE_EPOCH_DATE_STRING_WITH_OFFSET)));
+  }
+
+  @Test
+  public void testValidEmptyDuringRangeToXml() {
+    when(registry.lookup(eq(PROPERTY_NAME))).thenReturn(Optional.of(DATE_DESCRIPTOR));
+    assertThat(normalizer.normalizeForXml(PROPERTY_NAME, "/"), is(equalTo("/")));
```

Comment:
❓ Do we have a compelling reason to support this case? Why not just collapse it down to a blank string and forbid this entirely? Unless we need to support a "blank range", throwing an exception when we get this symbol on its own could help prevent issues down the road.

Reply:
This stems from the way we handle it in the front end. That being said, it isn't necessary to do a blank range. Since I want to be able to support forms with one or the other (e.g. "date"/"" or ""/"date") it seemed natural to allow for just the `/`. What issues are you concerned with?

Reply:
I don't think it'll be a huge deal. But we'll just have to support two values, `""` and `"/"`, that mean the same thing. Ideally we don't let how the UI operates dictate how the data gets persisted.